src/mmw/apps/geoprocessing_api/calcs.py

```diff
@@ -135,8 +135,33 @@ def catchment_water_quality(geojson):
             ]
         else:
             catchment_water_quality_results = []
+
+    def catchment_intersects_aoi(aoi, geom):
+        # Clamp minimum intersection percentage for the catchment between 1 and
+        # 5%, with the percentage sliding from 1 up to 5 if the AOI's > 1000
+        # sq km in size (deriving from the AOI sq km value / 1000).
+        aoi_kms = aoi.area / 1000000
+        min_intersection_pct = 1 if aoi_kms < 1000 else min(5, aoi_kms / 1000)
+        catchment_geom = GEOSGeometry(json.dumps(geom), srid=4326)
+        catchment_geom.transform(5070, clone=True)
```

Comment:
Since you're setting the `clone` arg, you'll need to capture the return variable.  I don't believe it mutates the object you're calling `transform` on.